contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtEventSequenceCollector.java

```diff
@@ -68,31 +68,50 @@ public class DrtEventSequenceCollector
 		PersonMoneyEventHandler, PersonDepartureEventHandler {
 
 	public static class EventSequence {
+
+		public final static class PersonEvents {
```

Comment:
Just wondering why not a record?

Reply:
@michalmac because fields in records are final and events are not added sequentially to the PersonEvents object